### PR TITLE
Show user friendly message when the plugin is not supported for installed platform

### PR DESCRIPTION
**Pull request by Fatme**

### CODE REVIEW

File: lib/services/plugins-service.ts [hunk 1]
```diff
@@ -67,7 +67,12 @@ export class PluginsService implements IPluginsService {
 				let pluginPlatformsData = pluginData.platformsData;
 				if(pluginPlatformsData) {
 					let pluginVersion = (<any>pluginPlatformsData)[platform];
-					if(semver.gt(pluginVersion, installedFrameworkVersion)) {
+					if(!pluginVersion) {
+						this.$logger.warn(`${pluginData.name} is not supported for ${platform}.`);
+						skipExecution = true;
```
Comment (teobugslayer) [inline lib/services/plugins-service.ts]: Can't you simply ```return ``` from the function?

Comment (Fatme) [inline lib/services/plugins-service.ts]: Yes, we can. I've just fixed it.

**Comment by teobugslayer** [comment]
:+1: after green build

**Comment by dtopuzov** [comment]
:+1:

**Pull request merged**